src/main/java/org/fundacionjala/at15/katas/pokerhands/diana/Flush.java

```diff
@@ -0,0 +1,16 @@
+package org.fundacionjala.at15.katas.pokerhands.diana;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Flush extends HandAnalysis {
+    private static final int SIX = 6;
+    protected int range;
```

Comment:
according to problem description, maybe `rank` is a more appropiate name for this var?

Reply:
I agree, I will change it.